src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1799,9 +1799,10 @@ private void UndoOrganelleMoveAction(MicrobeEditorAction action)
         if (!IsMoveTargetValid(newLocation, newRotation, organelle))
             return false;
 
-        // If it was already moved this session, it is free to move again
-        // Also free if not moved (but can be rotated)
-        int cost = (organelle.MovedThisSession || oldLocation == newLocation) ? 0 : Constants.ORGANELLE_MOVE_COST;
+        // If the organelle was already moved this session, added (placed) this session,
+        // or not moved (but can be rotated), then moving it is free
+        bool isFreeToMove = organelle.MovedThisSession || oldLocation == newLocation || organelle.PlacedThisSession;
```

Comment:
Variables should use `var` for consistency with other codes (and sometimes readability).

Reply:
I think it's fine to use bool here as it's a primitive type (this is the exception to the var rule mentioned in the style guide) and clarifies that the entire right side is a boolean expression